packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -296,11 +301,13 @@ class HotRunner extends ResidentRunner {
 
     // Picking up first device's compiler as a source of truth - compilers
     // for all devices should be in sync.
-    final List<Uri> invalidatedFiles = ProjectFileInvalidator.findInvalidated(
-      lastCompiled: flutterDevices[0].devFS.lastCompiled,
-      urisToMonitor: flutterDevices[0].devFS.sources,
-      packagesPath: packagesFilePath,
-    );
+    final List<Uri> invalidatedFiles =
```

Comment:
Avoid line breaks after an assignment operator: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters

Reply:
Fixed.